variants/frontend-base/js-lint/template.rb

```diff
@@ -19,7 +20,9 @@
   "format-check" => "prettier --check './**/*.{css,scss,json,md,js,ts,tsx,jsx}'",
   "format-fix" => "prettier --write './**/*.{css,scss,json,md,js,ts,tsx,jsx}'",
   "scss-lint" => "stylelint '**/*.{css,scss}'",
-  "scss-lint-fix" => "stylelint '**/*.{css,scss}' --fix"
+  "scss-lint-fix" => "stylelint '**/*.{css,scss}' --fix",
+  "test" => "jest /app/frontend",
+  "watch-tests" => "jest --watch"
```

Comment:
ditto for these lines regarding the `react` template variant.

Since this is a hash, you should be able to easily solve this:

```
package_json["scripts"] = {
  "js-lint" => "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx",
  "js-lint-fix" => "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts,tsx,jsx --fix",
  "format-check" => "prettier --check './**/*.{css,scss,json,md,js,ts,tsx,jsx}'",
  "format-fix" => "prettier --write './**/*.{css,scss,json,md,js,ts,tsx,jsx}'",
  "scss-lint" => "stylelint '**/*.{css,scss}'",
  "scss-lint-fix" => "stylelint '**/*.{css,scss}' --fix"
  "scss-lint-fix" => "stylelint '**/*.{css,scss}' --fix"
}

package_json.merge!({
  "test" => "jest /app/frontend",
  "watch-tests" => "jest --watch"
}) if is_using_jest
```

Reply:
this can be ignored per [this comment](https://github.com/ackama/rails-template/pull/216#discussion_r579915917)